### DISCUSSION TITLE
Add Context Window For Autocomplete like Cursor Editor

### DISCUSSION THREAD

**Discussion by yusufmumani** [zed-industries/zed#21624] [Feature Requests]
[https://news.ycombinator.com/item?id=41312932](url) Really makes sense
They’ve integrated autocomplete with LLM suggestions and it just works.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
